src/Compilers/VisualBasic/Portable/Parser/ParserFeature.vb

```diff
@@ -108,7 +108,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                     Return LanguageVersion.VisualBasic16_9
 
                 Case Feature.CallerArgumentExpression
```

Comment:
Is this actually checked anywhere?

Reply:
@333fred If you mean tests, there is `TestGoodCallerArgumentExpressionAttribute_Version16_9` that tests old version. If you mean in product code, it's checked here:

https://github.com/dotnet/roslyn/blob/db433641270916076a0b235c41db21da37860fdd/src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb#L3180-L3183